src/pages/app/index.tsx

```diff
@@ -53,13 +72,14 @@ const App = () => {
 
     const handleUrl = () => {
         if (!isLoggedIn) {
-            displayErrorMessage('Not logged in');
+            setShowLoginModal(true);
+            if (url) localStorage.setItem('url', url);
```

Comment:
why are we storing the url in local storage. can we not use a state variable for the same ?

Reply:
No , after login the callback url redirect to default page.